scipy/cluster/tests/test_hierarchy.py

```diff
@@ -769,7 +769,7 @@ def test_maxdists_one_cluster_linkage(self, xp):
         Z = xp.asarray([[0, 1, 0.3, 4]], dtype=xp.float64)
         MD = maxdists(Z)
         expectedMD = calculate_maximum_distances(Z, xp)
-        xp_assert_close(MD, expectedMD, atol=1e-15)
+        xp_assert_close(MD, expectedMD, rtol=1e-7, atol=1e-15)
```

Comment:
These are not needed if we do https://github.com/scipy/scipy/pull/20595/files#r1582009908